apollo-runtime/src/main/java/com/apollographql/apollo/internal/RealApolloWatcher.java

```diff
@@ -62,6 +62,10 @@
     cache.unsubscribe(recordChangeSubscriber);
   }
 
+  @Override public boolean isCanceled() {
+    return !isActive;
```

Comment:
In fact that's what I wanted to do in the first place, but decided against it because I didn't really want to change the existing code. But it makes more sense to change it to isCanceled.